src/lib/import_export/binary/binary_parser.cpp

```diff
@@ -185,21 +192,21 @@ std::shared_ptr<ValueSegment<T>> BinaryParser::_import_value_segment(std::ifstre
 template <typename T>
 std::shared_ptr<DictionarySegment<T>> BinaryParser::_import_dictionary_segment(std::ifstream& file,
                                                                                ChunkOffset row_count) {
-  const auto attribute_vector_width = _read_value<AttributeVectorWidth>(file);
+  const auto vector_compression_id = _read_value<VectorCompressionID>(file);
```

Comment:
Correct me if I'm wrong, but a `VectorCompressionID` represents the corresponding `CompressedVectorType`, right? The naming might be a bit misleading as one could assume that `VectorCompressionID` represents the corresponding `VectorCompressionType`.  How about using `CompressedVectorTypeID`?